app/frontend/src/features/communities/CommunityPage/DiscussionCard.tsx

```diff
@@ -1,37 +1,57 @@
-import {
-  Card,
-  CardActionArea,
-  CardContent,
-  makeStyles,
-  Typography,
-} from "@material-ui/core";
+import { Card, CardContent, makeStyles, Typography } from "@material-ui/core";
 import { Skeleton } from "@material-ui/lab";
 import classNames from "classnames";
-import CircularProgress from "components/CircularProgress";
-import { MORE_REPLIES } from "features/communities/constants";
-import useUsers, { useUser } from "features/userQueries/useUsers";
-import { Error as GrpcError } from "grpc-web";
+import Avatar from "components/Avatar";
+import { useUser } from "features/userQueries/useUsers";
 import { Discussion } from "pb/discussions_pb";
-import { GetThreadRes } from "pb/threads_pb";
-import { threadKey } from "queryKeys";
-import React, { useMemo } from "react";
-import { useInfiniteQuery } from "react-query";
+import { useMemo } from "react";
 import { Link } from "react-router-dom";
 import { routeToDiscussion } from "routes";
-import { service } from "service";
 import { timestamp2Date } from "utils/date";
-import { firstName } from "utils/names";
 import stripMarkdown from "utils/stripMarkdown";
 import { timeAgo } from "utils/timeAgo";
 
 const useStyles = makeStyles((theme) => ({
-  cardContent: { height: "100%" },
+  avatar: {
+    gridArea: "avatar",
+  },
+  cardContent: {
+    display: "flex",
+    "&&": {
```

Comment:
What does this do?

Reply:
It's a specificity hack (applying the class twice) needed to override MUI's `.MuiCardContent-root:last-child` rule. The ugly side of working with a framework when you want uniform padding and the framework decides to randomly add in more specific selectors with more padding to screw you over